Update the docs about the analysis metrics fields (#3677)
pipe-cd/pipecd: docs/content/en/docs-dev/user-guide/configuration-reference.md

|-|-|-|-|
| provider | string | The unique name of provider defined in the Piped Configuration. | Yes |
| strategy | string | The strategy name. One of `THRESHOLD` or `PREVIOUS` or `CANARY_BASELINE` or `CANARY_PRIMARY` is available. Defaults to `THRESHOLD`. | No |

> | query | string | A query performed against the [Analysis Provider](/docs/concepts/#analysis-provider). If there was no data point as a result of this query, that stage will be skipped. | Yes |

Reviewer: ```suggestion
| query | string | A query performed against the [Analysis Provider](/docs/concepts/#analysis-provider). The stage will be skipped if no data points were returned. | Yes |
```
